src/dataset-builder/DatasetBuilderUtils.test.ts

```diff
@@ -1,35 +1,35 @@
 import { div, span } from 'react-hyperscript-helpers';
 import {
   AnyCriteria,
-  AnyCriteriaApi,
   Cohort,
-  CohortApi,
   convertCohort,
   convertCriteria,
   convertDatasetAccessRequest,
   convertValueSet,
   CriteriaGroup,
-  CriteriaGroupApi,
   DatasetAccessRequest,
-  DatasetAccessRequestApi,
   DomainCriteria,
-  DomainCriteriaApi,
   formatCount,
   HighlightConceptName,
   ProgramDataListCriteria,
-  ProgramDataListCriteriaApi,
   ProgramDataRangeCriteria,
-  ProgramDataRangeCriteriaApi,
   ValueSet,
-  ValueSetApi,
 } from 'src/dataset-builder/DatasetBuilderUtils';
 import {
+  AnyCriteria as AnyCriteriaApi,
```

Comment:
In the yaml, this is called `SnapshotBuilderCriteria` (I think?), how would you feel about naming it as that? Or, if we need to have a situation where it is:
```
interface SnapshotBuilderCriteria {}
interface SnapshotBuilderDomainCriteria extends SnapshotBuilderCriteria {}
...
type {{NAME}} = SnapshotBuilderDomainCriteria | SnapshotBuilderProgramDataListCriteria | SnapshotBuilderProgramDataRangeCriteria;
```
then I would consider naming it `AnySnapshotBuilderCriteria`. How does that sound to you?

Reply:
This sounds good!